requirements.txt

```diff
@@ -1,7 +1,7 @@
 pyyaml>=3.0,<=3.99
 click>=6.0,<=6.99
 funcsigs==1.0.2 ; python_version == '2.7'
-future>-0.16,<=0.99 ; python_version == '2.7'
+future>=0.16,<=0.99 ; python_version == '2.7'
```

Comment:
Or maybe parse the requirements file to fill the `install_requires` and `extras_require` fields.